src/core/types.ts

```diff
@@ -71,6 +71,7 @@ export interface PolygonContracts extends BothContracts {
   ewQueue: EmergencyWithdrawalQueue;
   polygonDegen: StrategyVault;
   polygonLeverage: Vault;
+  affineGenesis: ethers.Contract;
```

Comment:
Should be AffineGenesis type.

Reply:
Where do I import that from? My inv throws an error that typechain doesnt have an exported member AffineGenmesis when I try to import